postgres-appliance/scripts/configure_spilo.py

```diff
@@ -338,7 +338,13 @@ def get_provider():
         return PROVIDER_LOCAL
 
     try:
-        logging.info("Figuring out my environment (Google? AWS? Openstack? Local?)")
+        logging.info("Figuring out my environment (Google? Azure? AWS? Openstack? Local?)")
+        # Check if Azure is the cloud provider
+        bashCommand='curl -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2019-03-11"'
+        c = os.popen(bashCommand).read()
+        cj = json.loads(c)
+        if cj["compute"]["provider"]=="Microsoft.Compute":
+            return PROVIDER_AZURE
```

Comment:
Since you are running on K8s you don't really need to detect the provider.

Reply:
Should I take this away?